servatrice/src/serversocketinterface.cpp

```diff
@@ -1000,3 +1002,108 @@ Response::ResponseCode ServerSocketInterface::cmdReloadConfig(const Command_Relo
     settingsCache->sync();
     return Response::RespOk;
 }
+
+Response::ResponseCode ServerSocketInterface::cmdAdjustMod(const Command_AdjustMod &cmd, ResponseContainer & /*rc*/) {
+
+    QString adminName = QString::fromStdString(cmd.admin_name());
+    QString userName = QString::fromStdString(cmd.user_name());
+    QList<ServerSocketInterface *> userList;    //list is used to allow functionality of sending event messages to multiple people if need be.
```

Comment:
I was thinking that the user may be logged in with different account names but the same client id.